src/test/java/com/joss/conductor/mobile/LocomotiveTest.java

```diff
@@ -250,55 +258,178 @@ public void test_get_center_window() {
                 .isEqualToComparingFieldByField(center);
     }
 
-    @Test
-    public void test_perform_swipe_center() {
+    private Map<String, List<Object>> getTouchActionParameters(TouchAction action)
+    {
+        try {
+            Method method = TouchAction.class.getDeclaredMethod("getParameters");
+            method.setAccessible(true);
+            return (Map)method.invoke(action);
+        } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    private void assertThatActionMatches(TouchAction actual, TouchAction expected) {
+        Map<String, List<Object>> actualParameters = getTouchActionParameters(actual);
+        Map<String, List<Object>> expectedParameters = getTouchActionParameters(expected);
+
+        assertThat((Map<String, List<Object>>)actualParameters, matchesEntriesIn((Map<String, List<Object>>)expectedParameters));
+    }
+
+    private void initMockDriver() {
+        initMockDriver(null);
+    }
+
+    private void initMockDriver(WebElement mockElement) {
```

Comment:
The method name is ambiguous, it implies that this method is initializes _all_ of the mockDriver mocks when it only adds window dimension and (if element not `@null`) size mocks.

Reply:
True, I'll adjust it.